test/integration/checkout_int_test.go

```diff
@@ -387,6 +387,26 @@ func (suite *CheckoutIntegrationTestSuite) TestNoLanguage() {
 	cp.ExpectExitCode(0)
 }
 
+func (suite *CheckoutIntegrationTestSuite) TestChangeSummary() {
+	if runtime.GOOS == "windows" {
+		suite.T().Skip("cp.SendCtrlC() does not work on Windows")
+	}
+	suite.OnlyRunForTags(tagsuite.Checkout)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/small-python"),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
+	)
+	cp.Expect("Resolving Dependencies")
+	cp.Expect("Done")
+	cp.Expect("Setting up the following dependencies:")
+	cp.Expect("└─ python@3.10.10")
+	cp.SendCtrlC() // don't care what happens after
+	cp.ExpectNotExitCode(0)
```

Comment:
Would be nice if we could check that only one dependency was listed, since we've had some bugs where we're giving too many.

We could potentially check if the terminal snapshot has only a single occurrence of `─`.